index.html

```diff
@@ -42,15 +42,12 @@
   <body data-cite="credential-management-1">
     <section id="abstract">
       <p>
-        This document specifies an API to allow user agents to mediate access
-        to, and representing, a verifiably-issued digital credential (e.g., a
-        government issued digital driver's license, a passport issued by a
-        country, a student card or diploma issued by a university, an employee
-        card issued by a company, a passenger's boarding pass issued by an
-        airline, a profile issued by an online social network, membership
-        cards, vaccination records, etc) license). The API builds on
-        [[[credential-management-1]]] as a means to request a digital
-        credential from the user agent or underlying platform.
+        This document specifies an API to enable [=user agents=] to mediate
+        access to, and present, a digital credential such as a driver's license,
+        a government-issued identification card, or [=credential type
```

Comment:
I think it reads better with all plural (or with all singular, but that rephrasing would be harder to write). Also, I *think* this API is meant to allow presenting multiple credentials (or selective-disclosure portions thereof), so `and/or`.
```suggestion
        access to, and present, digital credentials such as driver's licenses,
        government-issued identification cards, and/or [=credential type
```

Reply:
My only reservation was that generally people only have one driver's license, passport etc., but it's certainly possible that they could have multiple of other things. 

I don't mind though... @msporny, I'll leave the choice up to you.

Reply:
I attempted to address @marcoscaceres and @TallTed's concerns in 100094d89727d8b08368e16b94da0186b5a6b643.